gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/utils/DatasetUtils.java

```diff
@@ -44,20 +44,20 @@
     return Optional.<HivePartitionDataset>absent();
   }
 
-  public static String getProperty(HivePartitionDataset dataset, String property, int defaultValue) {
+  public static String getProperty(HivePartitionDataset dataset, String property, long defaultValue) {
```

Comment:
What is this used for? Shouldn't the return type of this method be long and the method be `getPropertyAsLong`?